package.json

```diff
@@ -30,6 +30,7 @@
     "bundle": "rollup -c",
     "bundle-types": "node bundle-types.js",
     "build": "npm run clean && npm run check-format && npm run lint && npm run compile && npm run docs && npm run bundle && npm run bundle-types",
+    "test:ci": "npm test -- --forbid-only ",
```

Comment:
Once this get merged in to master I can switch CI to run this instead

Reply:
Could you quickly describe what this does?

Reply:
I keep accidentally committing a `.only` in test files. This will error if any test file contains `.only` but the idea is to only have it run that way in CI. Locally you can `.only` all you want